### DISCUSSION TITLE
Russian Translation Zealot Robes

### DISCUSSION THREAD

**Discussion by Evolved-d** [FakeFishGames/Barotrauma#14950] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

After recent parch Zealot Robes got huge nerf - it is no longer stopping you from transformation. However, RU localisation is still stating it does.

### Reproduction steps

1. Start any campaign in RU Localisation Barotrauma 
2. Obtain Zealot Robes by console/outpost cleanse/trading
3. Observe the translation misleading me into transformation
//I will pin down screnshots later

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.6.17.0 (Unto the Breach Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! We'll report this to the localization team.

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/8ef1017c1ce425ffb08c5d7137f1b826ebd725df (commit in our private repo), fix coming up in the next update.
